# permissible values for mode

bool reset_pid_controller             # reset PID controller internal states.
bool reload_pid_from_ros              # reload PID gains from ROS param server.
RobotController/RobotCommand robot_command # unless reload_pid_from_ros is true,
                                      # values from robot_command will be read
                                      # into active PID controller.

---
bool success
string status_message
